### DISCUSSION TITLE
OTA flashing to a Sonoff Basic R1.

### DISCUSSION THREAD

**Discussion by GAZ082** [arendst/Tasmota#21784] [Q&A]
Hey there. In 2017 bought some sonoff basics r1 and flashed Tasmota by OTA, but now can't find instructions. Tried with the DIY mode but for some reason navigating to 10.10.7.1 (the gateway ip) does nothing. I did have some success by going to /devices, got a response with the APIKEY.

Thanks!

**Comment by Jason2866** [comment]
Forget and don't invest time in that horrible buggy Sonoff DIY mode. Get a cheap USB serial adapter and flash by "wire" (see Tasmota docu for details).

**Reply by GAZ082** [reply]
Hi Jason, thanks! But I was able to easily flash 3 devices with who knows what instructions, 6 years ago. Should be able to do the same if i just find the instructions!

If this serial method does not require any soldering, just plugging the connector, i may give a chance. But I dont know if it's that easy with the R1.

**Reply by Jason2866** [reply]
Sorry can't help with the DIY mode. Maybe someone reads this who has a working workflow documentation
